frontend/amundsen_application/static/js/components/Alert/styles.scss

```diff
@@ -5,9 +5,9 @@
 @import 'typography';
 
 $alert-border-radius: 4px;
-$alert-message-line-height: 24px;
 $alert-warning-background: #fff0d4;
 $alert-alert-background: #ffe4dd;
+$alert-element-margin: auto 0 auto auto;
```

Comment:
the top and bottom margins look good in the screenshot now, but the message is a little far over from the icon IMO. maybe we can have two separate margin values here, this one you have used for the alert-action, then `auto auto auto 0` used for the alert-message so that we can have 0 on the left side like before